packages/fxa-payments-server/src/App.tsx

```diff
@@ -50,39 +53,77 @@ export const App = ({
     locationReload,
   };
   return (
-    <StripeProvider apiKey={config.stripe.apiKey}>
-      <ReduxProvider store={store}>
-        <AppContext.Provider value={appContextValue}>
-          <Router>
-            <React.Suspense fallback={<RouteFallback />}>
-              {/* Note: every Route below should also be listed in INDEX_ROUTES in server/lib/server.js */}
-              <Route
-                path="/"
-                exact
-                render={() => <Redirect to="/subscriptions" />}
-              />
-              <Route
-                path="/subscriptions"
-                exact
-                render={props => (
-                  <SettingsLayout>
-                    <Subscriptions {...props} />
-                  </SettingsLayout>
-                )}
-              />
-              <Route
-                path="/products/:productId"
-                render={props => (
-                  <SignInLayout>
-                    <Product {...props} />
-                  </SignInLayout>
-                )}
-              />
-            </React.Suspense>
-          </Router>
-        </AppContext.Provider>
-      </ReduxProvider>
-    </StripeProvider>
+    <AppContext.Provider value={appContextValue}>
```

Comment:
Indentation changes really obscure this change: wrapped everything under `<StripeProvider>` in a new `<AppErrorBoundary>` component that will trap exceptions and render `AppErrorDialog`.

But since `AppErrorBoundry` needs access to the `appContext`, I moved `<AppContext.Provider>` to the very top of the composition stack.